src/components/HeroBlock/HeroBlock.jsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import HeroBlockComponent from "./HeroBlockComponent";
+import HeroBlockPicture from "./img/heroBlockPicture.png";
+
+function HeroBlock() {
+  return (
+    <HeroBlockComponent
+      title="Izbaudi profesijas ēnošanas piedzīvojumu un atklāj savu karjeras
+  potenciālu!"
+      subtitle_span="TryWork"
```

Comment:
It's not correct props name.
Mostly in React you do props name with Camel case, not snake case.